styles.css

```diff
@@ -42,6 +42,74 @@ nav {
   width: 40px;
 }
 
+.hidden {
+  display: none;
+}
+
+#btn-close {
+  border: none;
+  background-color: #6070ff00;
+  color: #fff;
+  cursor: pointer;
+  font-size: 24px;
+  position: fixed;
+  right: 70px;
+  z-index: 3;
+}
+
+.menu-toggle {
+  animation-name: slidein;
+  animation-duration: 1s;
+  color: #fff;
+  display: block;
+  font-size: 32px;
+  font-weight: 600;
+  position: fixed;
+  left: 28px;
+  top: 50px;
+  z-index: 5;
+}
+
+.menu-toggle > li > a {
+  color: #fff;
+  margin: 20px 0;
+}
+
+.overlay {
+  animation-name: slidein;
+  animation-duration: 1s;
+  background-color: #0517be8f;
+  backdrop-filter: blur(10px);
+  position: fixed;
+  height: 100vh;
+  width: 100vw;
+  z-index: 2;
+}
+
+@keyframes slidein {
+  from {
+    margin-left: 100%;
+    width: 300%;
+  }
+
+  to {
+    margin-left: 0%;
+    width: 100%;
+  }
+}
+
+@keyframes slideout {
+  from {
+    margin-left: 0%;
+    width: 100%;
+  }
+
+  to {
+    margin-left: 100%;
+    width: 300%;
+  }
+}
+
```

Comment:
- [ ] Good job so far your menu works as expected :clap:  but we have an issue with the design :cry: . Please try to work on your `Xmark ` and also your menu option about/contact/portfolio spacing margin/padding in order to make them similar to the Figma template. Note that it's important to pay attention to details as a developer :+1: 

### Figma design
![Screenshot_20221219_153157](https://user-images.githubusercontent.com/6615394/208464060-e7c698de-e73e-4917-996e-c7bd962aa76b.jpg)

### Your design
![Screenshot_20221219_153405(1)](https://user-images.githubusercontent.com/6615394/208464068-56bfc40c-f059-4915-b79b-0efff901952b.jpg)